docs/src/examples/components/Label/Variations/LabelExampleOnIconClick.shorthand.tsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react'
 import { Label } from '@stardust-ui/react'
 
-class LabelExampleIconShorthand extends React.Component<{}, { display: string }> {
+class LabelExampleIconShorthand extends React.Component<any, any> {
```

Comment:
No typings in examples, please.  They should be geared for minimalism and all consumers.